deploy/s3_deploy.sh

```diff
@@ -2,10 +2,6 @@
 
 # this will deploy the current public folder to a subfolder in the s3 bucket
 # the subfolder is the name of the TRAVIS_BRANCH
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-	echo "skiping deploy to S3: this is a pull request"
-	exit 0
-fi
```

Comment:
I'm curious about this change. Did you change the Travis settings? (There's a Travis setting that I've wondered about whether it would obviate the need for this.) Have you tested whether both pull request and branch builds are deployed?